src/components/HeroIcon.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './HeroIcon.scss';
+
+const HeroIcon = ({ src, alt }) => (
+  <div className="ins-c-hero__title-icon">
```

Comment:
This is using flex layout right? Might be more reasonable to use the layout
```JSX
import { Flex, FlexItem } from '@patternfly/react-core';
const HeroIcon = (props) => (
  <Flex className="ins-c-hero__title-icon">
    <img {...props} aria-hidden="true" />
  </Flex>
);
```

The only thing that will be still required is the `width`.